tests/providers/aws/services/organizations/organizations_account_part_of_organizations/organizations_account_part_of_organizations.py

```diff
@@ -0,0 +1,60 @@
+from unittest import mock
+
+from boto3 import client
+from moto import mock_organizations
+
+AWS_REGION = "us-east-1"
+
+
+class Test_organizations_account_part_of_organizations:
+    @mock_organizations
+    def test_no_organization(self):
+        from prowler.providers.aws.lib.audit_info.audit_info import current_audit_info
```

Comment:
We are changing this because is better to mock the `current_audit_info` object every time we use it in tests so use the `mock_set_audit_info` instead.

Reply:
At the beginning we started importing the  `current_audit_info` object into the check's test but we realised that it was a bad practice since if shared among the code and for the tests is better to be in an isolated scenario. What I said is to use the `set_mock_audit_info`:
```python
# Mocked Audit Info
    def set_mocked_audit_info(self):
        audit_info = AWS_Audit_Info(
            session_config=None,
            original_session=None,
            audit_session=session.Session(
                profile_name=None,
                botocore_session=None,
            ),
            audited_account=None,
            audited_user_id=None,
            audited_partition="aws",
            audited_identity_arn=None,
            profile=None,
            profile_region=None,
            credentials=None,
            assumed_role_info=None,
            audited_regions=None,
            organizations_metadata=None,
            audit_resources=None,
        )
        return audit_info
```

So, if you can change the test you did and instead of importing the shared object create a mocked one.

Reply:
fixed 🚀